Moya-Gloss.podspec

```diff
@@ -8,7 +8,7 @@
 
 Pod::Spec.new do |s|
   s.name             = "Moya-Gloss"
-  s.version          = "2.0.0-beta.2"
+  s.version          = "2.0.0-beta.3"
```

Comment:
can you keep the version number the same? 
I'm in the middle of managing new version updates and I'll handle that little bit :)